TribalWars/src/ApplicationCore/Results/Generic/ServiceResult.cs

```diff
@@ -7,14 +8,21 @@ public class ServiceResult<T> : ServiceResult
     {
         public T Content { get; }
 
-        public ServiceResult(ServiceResultStatus serviceStatus, IEnumerable<string> errors, T content) : base(serviceStatus, errors)
+        private ServiceResult(ServiceResultStatus serviceStatus, IEnumerable<string> errors, T content) : base(serviceStatus, errors)
         {
             Content = content;
         }
+        private ServiceResult(ServiceResultStatus serviceStatus, IEnumerable<string> errors) : base(serviceStatus, errors)
+        {
+        }
 
         public static ServiceResult<T> Success(T content)
         {
             return new(ServiceResultStatus.Success, null, content );
         }
+        public static new ServiceResult<T> Failure(string error)
+        {
+            return new(ServiceResultStatus.Failure, new List<string> { error });
+        }
```

Comment:
This doesn't relate to the T type, so it acts the same as normal `ServiceResult Failure(string error)` method